README.md

```diff
@@ -31,6 +31,7 @@ If there is an issue with the database necessitating a re-release within the mon
 * .NET (C#): [Sepp Wijnands] [swijnands] at [iPerform Software] [iperform]
 * PHP: [Lars Strojny] [lstrojny]
 * Go: [Thomas Sileo] [tsileo]
+* Erlang [Silviu Caragea] [silviucpp]
```

Comment:
You can remove the space between `[Silviu Caragea]` and `[silviucpp]` otherwise the link won't work

Reply:
Hmm from what I see links are working fine.. and also all other are like this in readme. 

You can test this here: https://github.com/silviucpp/referer-parser

Reply:
yeah no what I mean is that the link will be rendered as [Silviu Caragea] [silviucpp](https://github.com/silviucpp) instead of [Silviu Caragea](https://github.com/silviucpp), the markdown spec changed a while back and this readme hasn't been updated.